webapp/packages/plugin-data-viewer/src/TableViewer/ValuePanel/Presentation/BooleanValue/useValuePanelBooleanValue.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2024 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+import { ResultSetSelectAction } from '../../../../DatabaseDataModel/Actions/ResultSet/ResultSetSelectAction';
+import { ResultSetViewAction } from '../../../../DatabaseDataModel/Actions/ResultSet/ResultSetViewAction';
+import type { IDatabaseDataModel } from '../../../../DatabaseDataModel/IDatabaseDataModel';
+import type { IDatabaseResultSet } from '../../../../DatabaseDataModel/IDatabaseResultSet';
+import { isStringifiedBoolean } from './isBooleanValuePresentationAvailable';
+
+export function useValuePanelBooleanValue(model: IDatabaseDataModel<any, IDatabaseResultSet>, resultIndex: number): boolean | null | undefined {
```

Comment:
Please check usage. You already have active elements where you use this hook. Maybe it would be better to replace it with a function that will only transform the given value.